Packs/FeedOpenCTI/Integrations/FeedOpenCTI_v2/FeedOpenCTI_v2_description.md

```diff
@@ -0,0 +1,5 @@
+## Configure OpenCTI Feed v2 on Cortex XSOAR
+
+1. Enter **Base URL8** and **API Key**.
```

Comment:
not sure i understand why this is needed here

Reply:
changed